README.md

```diff
@@ -23,10 +24,10 @@ source .venv\Scripts\activate.bat
 note! The python executable might be different on your computer. If this doesn't work, you may need to use `python` or `py` instead of `python3`
```

Comment:
Can you use a note section to highlight this?

https://github.com/orgs/community/discussions/16925